src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -179,6 +191,7 @@ const useGeneViewRouting = () => {
   const { search } = useLocation();
   // TODO: discuss – is using URLSearchParams better than using the querystring package?
   const view = new URLSearchParams(search).get('view');
+  const transcriptId = new URLSearchParams(search).get('transcriptId');
```

Comment:
shouldn't need to create `new URLSearchParams(search)` twice.  `urlSearchParams = new URLSearchParams(search); const view = urlSearchParams.get('view'); const transcriptId = urlSearchParams.get('transcript_id)`

Reply:
That's done